multiworld/envs/pygame/point2d.py

```diff
@@ -177,11 +190,16 @@ def sample_goals(self, batch_size):
                 0
             )
         else:
-            goals = np.random.uniform(
-                self.obs_range.low,
-                self.obs_range.high,
-                size=(batch_size, self.obs_range.low.size),
-            )
+            goals = np.zeros((batch_size, self.obs_range.low.size))
+            for b in range(batch_size):
+                goals[b, :] = self._sample_position(self.obs_range.low,
+                                         self.obs_range.high,)
+                                         # realistic=self.sample_realistic_goals)
```

Comment:
are we going to assume all the goals we are sampling are going to be realistic? In my code I had this as an input to the constructor

Reply:
Hmmm... I don't see a big use-case for purposefully generating unrealistic goals (other than for debugging)